### PR TITLE
Add IO version of `decodeXls` function.

### PR DESCRIPTION
**Pull request by nixorn**
Thanks for library!
In our situation we do not need `Conduit` wrapping, it is hard to integrate it. 
Also files we have is relatively small, and we need to parse couple per day.

### CODE REVIEW

File: lib/Data/Xls.hs [hunk 1]
```diff
@@ -104,6 +110,28 @@ decodeXls file =
...
+
+-- | Parse a Microsoft excel xls workbook file into a [[[String]]].
+-- Nesting levels means worksheets rows and cells accordingly.
+--
+-- Throws 'XlsException'
+--
+decodeXlsIO
+  :: FilePath
+  -> IO [[[String]]]
+decodeXlsIO file = do
```
Comment (harendra-kumar) [inline lib/Data/Xls.hs:122]: Let's keep the indentation consistent as 4 spaces everywhere.

**Comment by harendra-kumar** [comment]
Looks good. Please add a changelog entry as well describing the enhancement made.

Did you have any trouble with the conduit API?

I have not added any tests, it will be nice if you can commit a sample xls file and add a simple test to sanity check if everything is working correctly.

You can send another PR, bumping the version of the package to 0.1.3. If you would like, I can add you as a maintainer as well so that you can upload the new version to Hackage yourself.

File: lib/Data/Xls.hs (hunk 1 above)
Comment (nixorn) [reply lib/Data/Xls.hs:122]: Done.

**Comment by nixorn** [comment]
The main trouble with conduit - I did not figure out how to get parsed values back to IO monad.
About tests - ore likely I'll add tests on monday.

**Comment by nixorn** [comment]
Done with tests.

**Comment by nixorn** [comment]
You can add me as a maintainer if you want. I'll upload updates.

**Comment by nixorn** [comment]
Discovered corner case during battle-testing.
Xls with characters in local windows encoding (cp1251 for Russia) being decoded as `String` can produce nasty errors, like corrupted JSON from Aeson `encode`. `encode` works well, but browser can not parse json.
I going to use ByteString instead of String as result of parse, to let user decide how to decode himself.

**Comment by nixorn** [comment]
Never mind, corrupted json is local bug on my side. Need to figure out how to fix this windows-based CI. I committed `test/data/test.xls`, but CI don't see it, probably base directory in CI not equal to project root, or CI does not mount directory...

`stack test` works well locally. Can you help with this?

**Comment by harendra-kumar** [comment]
You need add this file to `extra-source-files` section in the cabal file. The CI builds the package from a distribution tarball, if you have not packaged this file then it will not be able to find it.

**Comment by nixorn** [comment]
CI is green.
